<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>160</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pomodoro</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>tomate.png</normaloff>tomate.png</iconset>
  </property>
  <widget class="QPushButton" name="bt_iniciar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(0, 255, 0);
	color: rgb(0, 0, 0);
	border-radius: 10px
}

QPushButton:pressed{	
	background-color: rgb(170, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_tempo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>56</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Tempo:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_cronometro">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>340</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>340</width>
     <height>31</height>
    </size>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(188, 188, 188);
</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_parar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	color: rgb(255, 255, 255);
	background-color: rgb(255, 0, 0);
	border-radius: 10px
}

QPushButton:pressed{	
	
	background-color: rgb(170, 85, 0);
}</string>
   </property>
   <property name="text">
    <string>Parar</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>bt_iniciar</zorder>
  <zorder>ln_tempo</zorder>
  <zorder>pb_cronometro</zorder>
  <zorder>label_2</zorder>
  <zorder>bt_parar</zorder>
 </widget>
 <resources>
  <include location="tomate.qrc"/>
 </resources>
 <connections/>
</ui>
